### PR TITLE
Add navigation 

### PR DESCRIPTION
**Pull request by OstapivIvan**
Added navigation with logo, links and adaptive styles

### CODE REVIEW

**Comment by OstapivIvan** [comment]
https://github.com/proga-react-practice/sigma-team-project/assets/123650171/647a43d7-e034-4cea-981e-23f2e8e1b110


https://github.com/proga-react-practice/sigma-team-project/assets/123650171/b46a538e-6b73-48ac-8cfa-b6e2c538465d

Navigation

**Review by OliaZakharcuk — COMMENTED** [review]
Can you center this? 
image

**Comment by OliaZakharcuk** [comment]
Add some space and make them the same size
image

**Comment by OliaZakharcuk** [comment]
replace with team contacts or drop as you already have logo on the other side
image

File: src/components/Header.tsx [hunk 1]
```diff
@@ -1,116 +1,269 @@
...
+    const contacts = ["Telegram", "Instagram"];
 
-            </Badge>
-          </Toolbar>
-        </AppBar>
-      </Box>
-    </ThemeProvider>
-  );
+    return (
+        <ThemeProvider theme={theme}>
```
Comment (OliaZakharcuk) [inline src/components/Header.tsx:64]: move to the highest level

File: src/components/Header.tsx [hunk 2]
```diff
@@ -1,116 +1,269 @@
...
+            <Box sx={{flexGrow: 1}}>
+                <AppBar
+                    position="static"
+                    sx={{
+                        "@media screen and (max-width: 426px)": {
```
Comment (OliaZakharcuk) [inline src/components/Header.tsx:69]: try to use mui breakpoints

File: src/components/Header.tsx [hunk 3]
```diff
@@ -1,116 +1,269 @@
...
+                                    sx={{p: 0}}
+                                >
+                                    <Avatar
+                                        alt="Bohdan Mylyi"
+                                        src="./src/images/photo_2024-03-31_18-21-22.jpg"
+                                    />
+                                </IconButton>
+                            </Tooltip>
+                            <Menu
+                                sx={{mt: "45px"}}
```
Comment (OliaZakharcuk) [inline src/components/Header.tsx:224]: no hardcode

File: src/components/Header.tsx [hunk 4]
```diff
@@ -1,116 +1,269 @@
...
+                                        position: "absolute",
+                                        top: 0,
+                                        left: 0,
+                                        width: "100%",
+                                        height: "100%",
+                                        borderRadius: "50%",
+                                        animation:
+                                            "ripple 1.2s infinite ease-in-out",
+                                        border: "1px solid currentColor",
+                                        content: '""',
```
Comment (OliaZakharcuk) [inline src/components/Header.tsx:197]: Why is it needed?

**Review by OliaZakharcuk — CHANGES_REQUESTED** [review]

File: src/pages/Home.tsx [hunk 5]
```diff
@@ -1,31 +1,62 @@
...
+    return (
+        <Container 
+        sx={{
+            textAlign: "center",
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+            height: '100vh',
+        }}>
+            <Container sx={{padding: 0}}>
```
Comment (OliaZakharcuk) [inline src/pages/Home.tsx:15]: Why do you need two containers?

**Review by OliaZakharcuk — APPROVED** [review]

**Pull request merged**